cookiecutter-django-app/README.rst

```diff
@@ -141,43 +139,3 @@ Add to Django Packages
 ======================
 
 Once you have a release, and assuming you have an account there, just go to https://www.djangopackages.com/packages/add/ and add it there.
-
```

Comment:
Why are you removing this text from the README? Was this intentional?

Reply:
Oh, perhaps the answer is, "Because this is the readme for the cookiecutter, and this is covered by the top-level readme."

Reply:
Yeah, exactly. (I mentioned this in one of the commits, but I haven't yet updated the PR description to roll up all the change descriptions... need to do that.)